tests/tree_util_test.py

```diff
@@ -425,5 +425,86 @@ def testDtypeMonomorphicUnravel(self):
       _ = unravel(y)
 
 
+class TreePrefixErrorsTest(jtu.JaxTestCase):
+
+  def test_different_types(self):
```

Comment:
Shouldn't there be some tests for no errors? 
E.g., (1, 2), ((11, 12, 13), 2)

Reply:
Done!